packages/extension/src/companion/Companion.tsx

```diff
@@ -113,16 +107,6 @@ export default function Companion({
       companionExpanded={companionState}
       shouldLoad={assetsLoaded}
     >
-      {!user && !loadingUser && shouldShowLogin && (
-        <LoginModal
-          parentSelector={getCompanionWrapper}
-          isOpen
-          onRequestClose={closeLogin}
-          contentLabel="Login Modal"
-          {...loginState}
-        />
-      )}
```

Comment:
Do we have a separate instance of the AuthModal in our Companion?

Reply:
Companion no longer has the modal, it goes directly to the designated `signup` page.